.github/workflows/release.yml

```diff
@@ -38,23 +38,53 @@ jobs:
           python .ci-helpers/get_next_version.py
           echo "NEW_TAG=$(python .ci-helpers/get_next_version.py)" >> $GITHUB_ENV
 
+      - name: Initialize release
+        uses: softprops/action-gh-release@v1
+        with:
+          name: TARDIS v${{ env.NEW_TAG }}
+          tag_name: release-${{ env.NEW_TAG }}
+          body: "This release has been created automatically by the TARDIS continuous delivery pipeline."
+          draft: false
+      
+      - name: Wait for Zenodo to update the latest release
+        run: sleep 30
```

Comment:
Maybe this should be longer just to be safer in allowing zenodo enough time for the DOI to generate.

Reply:
Agreed. 1 minute or more would be safer.

Reply:
I updated it to 120 seconds.